modules/sdk-coin-bsc/src/index.ts

```diff
@@ -0,0 +1,3 @@
+export * as BscLib from './lib';
```

Comment:
Undocumented, but we're going go export everything without the name. I need to change this in the sdk-coin:new script.

Reply:
ah ok, will do right now.